src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -1,11 +1,21 @@
 package core.basesyntax.dao;
 
 import core.basesyntax.db.Storage;
-import core.basesyntax.model.Fruit;
+import java.util.Map;
 
 public class FruitDaoImpl implements FruitDao {
     @Override
-    public void add(Fruit fruit, int amount) {
-        Storage.fruits.put(fruit, Storage.fruits.get(fruit) + amount);
+    public void add(String nameFruit, int amount) {
+        Storage.fruits.put(nameFruit, Storage.fruits.get(nameFruit) + amount);
```

Comment:
There should not be any logic in DAO. Amount in parameters should be counted before.